packages/css-framework/src/components/_container.scss

```diff
@@ -1,24 +1,39 @@
 .rn-container {
-  padding-left: 40px;
-  padding-right: 40px;
+  max-width: $navy-content-width;
+  margin: 0 auto;
+  padding: spacing(3);
```

Comment:
Have updated the padding for the container to only be on left and right, but have not gone so far as to have a separate div to set the vertical padding otherwise we start to get tags purely to apply extra styles. I think the new code is a good balance.